.kokoro/release/common.cfg

```diff
@@ -67,3 +67,83 @@ env_vars: {
 
 ###################################################
 
+
+
+#############################################
+# this section merged from .kokoro/common_env_vars.cfg using owlbot.py
+
+env_vars: {
```

Comment:
This is being continually added by your owlbot.py post-processor. Anything in the post-processor needs to be idempotent.